docs/interactions/Receiving_and_Responding.md

```diff
@@ -209,22 +209,22 @@ There are a number of ways you can respond to an interaction:
-| MODAL\*\*                               | 9     | respond to an interaction with a popup modal                                                                  |
-| PREMIUM_REQUIRED\*\*\*                  | 10    | respond to an interaction with an upgrade button, only available for apps with monetization enabled           |
+| Name                                    | Value | Description                                                                                                                                               |
+|-----------------------------------------|-------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| PONG                                    | 1     | ACK a `Ping`                                                                                                                                              |
+| CHANNEL_MESSAGE_WITH_SOURCE             | 4     | respond to an interaction with a message                                                                                                                  |
+| DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE    | 5     | ACK an interaction and edit a response later, the user sees a loading state                                                                               |
+| DEFERRED_UPDATE_MESSAGE\*               | 6     | for components, ACK an interaction and edit the original message later; the user does not see a loading state                                             |
+| UPDATE_MESSAGE\*                        | 7     | for components, edit the message the component was attached to                                                                                            |
+| APPLICATION_COMMAND_AUTOCOMPLETE_RESULT | 8     | respond to an autocomplete interaction with suggested choices                                                                                             |
+| MODAL\*\*                               | 9     | respond to an interaction with a popup modal                                                                                                              |
+| PREMIUM_REQUIRED\*\*\*                  | 10    | [Deprecated](#DOCS_CHANGE_LOG/premium-interactions). respond to an interaction with an upgrade button, only available for apps with monetization enabled  |
 
 \* Only valid for [component-based](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/) interactions
 
 \*\* Not available for `MODAL_SUBMIT` and `PING` interactions.
 
-\*\*\* Not available for `APPLICATION_COMMAND_AUTOCOMPLETE` and `PING` interactions.
+\*\*\* This response type is deprecated. Learn more about [migrating to premium buttons](#DOCS_MONETIZATION_APP_SUBSCRIPTIONS/gating-premium-interactions). Not available for `APPLICATION_COMMAND_AUTOCOMPLETE` and `PING` interactions.
```

Comment:
> Maybe this can all look neater too

I'd agree that putting it in the footnote isn't optimal. Maybe put the deprecation note in a warn box instead...

Reply:
> this may confuse developers that might think that PING/PONG/APPLICATION_COMMAND_AUTOCOMPLETE/APPLICATION_COMMAND_AUTOCOMPLETE_RESULT are deprecated.

Made a small change that hopefully makes it more clear. be2df07187315a440cf482a7ce980b6e60f2a0df